crates/model/src/solver_competition.rs

```diff
@@ -109,13 +109,30 @@ pub struct SolverSettlement {
     pub uninternalized_call_data: Option<Vec<u8>>,
 }
 
+#[serde_as]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(untagged)]
+pub enum Cost {
```

Comment:
Will we be able to differentiate in the swagger endpoint and logs whether a solution contained solver or protocol estimates?

Reply:
Yes, it should be serialised with, for example, `cost: Protocol(5.6)`